exercises/isogram/test/test_isogram.c

```diff
@@ -15,6 +15,12 @@ void test_empty_string(void)
    TEST_ASSERT_TRUE(is_isogram(""));
 }
 
+void test_null(void)
+{
+   TEST_IGNORE();
+   TEST_ASSERT_FALSE(is_isogram(NULL));
+}
+
 void test_lower_case_only(void)
 {
    TEST_IGNORE();               // delete this line to run test
```

Comment:
Can you move `// delete this line to run test` to the new test? We put this on the second test of all exercises.

Reply:
Since you have your settings configured to allow me to make changes on your pull requests for this repository (thanks!) I went ahead and made this small change.